### PR TITLE
docs: Docker image versions

### PR DESCRIPTION
**Pull request by boc-the-git**
## What type of PR is this?

- documentation

## What this PR does / why we need it:

Make clear the suggestion to use a specific version tag, rather than "latest" for docker images.

## Which issue(s) this PR fixes:

None.

### CODE REVIEW

File: docs/docs/contributors/developers-guide/maintainers.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Mealie is published via GitHub actions to the GitHub container registry with the
 : published when a new GitHub Release is created - [Actions File](https://github.com/mealie-recipes/mealie/blob/mealie-next/.github/workflows/release.yml)
 
 !!! note
-    Both the latest, and {version} tags will be the same container on the release of a new version
+    Both the latest, and {version} tags will be the same image on the release of a new version (until the next nightly is published)
```
Comment (Kuchenpirat) [inline docs/docs/contributors/developers-guide/maintainers.md]: Should latest not be the latest release? So nightly is not equal to latest.

Comment (boc-the-git) [reply docs/docs/contributors/developers-guide/maintainers.md]: Brain fart after a long day.. Will remove the bit in brackets.

Comment (boc-the-git) [reply docs/docs/contributors/developers-guide/maintainers.md]: Good catch. It made sense in my head as I wrote it.. I was trying to say there will be releases (on nightly) that are need than latest or {version}

File: docs/docs/documentation/getting-started/updating.md [hunk 2]
```diff
@@ -20,5 +20,6 @@ If you are upgrading from pre-v1.0.0 to v1.0.0, make sure you read [Migrating to
 For all setups using Docker the updating process looks something like this
 
 - Stop the container using `docker compose down`
+- Optionally change the version (image tag) specified in your docker-compose file
```
Comment (Kuchenpirat) [inline docs/docs/documentation/getting-started/updating.md]:
```suggestion
- If you are not using the latest tag, change the version (image tag) in your docker-compose file
```

I guess something like that would be more precise.

**Review by Kuchenpirat — APPROVED** [review]
LGTM 👍

**Pull request merged**